client/src/app/app.component.html

```diff
@@ -14,6 +14,7 @@
   <button mat-button *ngIf="hasRole([UserRole.moderator])" [matMenuTriggerFor]="testMenu">Tests</button>
   <mat-menu #testMenu="matMenu">
     <button mat-menu-item (click)="onTestListClick()">Tests list</button>
+    <button mat-menu-item (click)="onRedactorTestsListClick()">Redactor's test list</button>
```

Comment:
To podmenu powinno być przeniesione w inne miejsce, albo `*ngIf` powinien być przeniesiony. W tym wypadku redaktor nie zobaczy tego menu, ponieważ wyżej jest `*ngIf="hasRole([UserRole.moderator])"`.